modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp

```diff
@@ -336,6 +336,8 @@ renderResponse.setTitle(blogsEditEntryDisplayContext.getPageTitle(resourceBundle
 								modelName="<%= BlogsEntry.class.getName() %>"
 							/>
 						</aui:fieldset>
+
+						<aui:input name="showPermissions" type="hidden" value="<%= true %>" />
```

Comment:
Can you rename the parameter to `addEntryResources`. In the backend it is confusing (it is not showing anything).